airflow-core/src/airflow/dag_processing/bundles/manager.py

```diff
@@ -292,3 +293,9 @@ def view_url(self, name: str, version: str | None = None) -> str | None:
         )
         bundle = self.get_bundle(name, version)
         return bundle.view_url(version=version)
+
+    @staticmethod
+    def get_dagbag(**kwargs) -> DagBagType:
```

Comment:
I'm a bit torn on this method to be honest. Feels like if we are going to do something like this, it should just give you a bag for a specific bundle (e.g. `manager.get_dagbag(bundle=somebundle)` or `manager.get_dagbag(bundle_name="somename")`).

I think I'm in favor of just letting folks continue to use DagBag directly like they do today though. e.g.

```
bag = DagBag(bundle.path, bundle_path=bundle.path)
```